### ISSUE TITLE
Can't upgrade due to .Net Standard dependency

### ISSUE THREAD

**Issue by ramedey** [Viincenttt/MollieApi#55]
I got stuck with an older version because my solution does not support .Net Standard. 

Would it be possible to add a dependency group in the nuspec file that targets .NET Framework 4.5 without .Net Standard?

I believe it would be something like this (not 100% sure though):
```xml
    <dependencies>
      <group targetFramework=".NETFramework4.5">
         <dependency id="Newtonsoft.Json" version="10.0.3" exclude="Build,Analyzers" />
      </group>
      <group targetFramework=".NETStandard1.2">
        <dependency id="NETStandard.Library" version="1.6.1" exclude="Build,Analyzers" />
        <dependency id="Newtonsoft.Json" version="10.0.3" exclude="Build,Analyzers" />
      </group>
    </dependencies>
```

**Comment by Sebazzz** [comment]
No, you'd need to multitarget, like this:

    <TargetFrameworks>netstandard1.2;net45</TargetFrameworks>

**Comment by Viincenttt** [comment]
Fixed, thanks @Sebazzz

**Comment by Viincenttt** [comment]
The issue should now be resolved in version 2.0.3.0, which is available for download.
